Unable to use fabric:2.4 binding in conjunction with docker image

When using the latest unstable docker image `0.5.0-unstable-20220512172343` binding to fabric 2.4 completes but transaction requests fail with 

```
2022.05.13-11:53:55.532 info  [caliper] [caliper-engine]        Network configuration attribute "caliper.command.start" is not present, skipping start command
2022.05.13-11:53:56.045 info  [caliper] [FabricConnectorFactory]        Initializing peer gateway connector compatible with installed fabric-gateway SDK: 1.0.1
2022.05.13-11:53:56.046 error [caliper] [caliper-engine]        Error while performing "test" step: Error: Cannot find module '@grpc/grpc-js'
```

We need to explicitly bind grpc-js

--- a/packages/caliper-cli/lib/lib/config.yaml
+++ b/packages/caliper-cli/lib/lib/config.yaml
@@ -43,7 +43,7 @@ sut:
             packages: ['fabric-network@2.2.12']
         2.2: *fabric-v2-lts
         2.4: 
-            packages: ['@hyperledger/fabric-gateway@1.0.1']
+            packages: ['@hyperledger/fabric-gateway@1.0.1', '@grpc/grpc-js@1.6.7']
         latest: *fabric-v1-lts
         latest-v2-lts: *fabric-v2-lts
         latest-v2: *fabric-v2-lts

